modules/PreviewAPI/src/main/java/org/gephi/preview/PreviewModelImpl.java

```diff
@@ -82,8 +80,7 @@ Development and Distribution License("CDDL") (collectively, the
     //Properties
     private PreviewProperties properties;
     //Dimensions
-    private Dimension dimensions;
-    private Point topLeftPosition;
+    private Rectangle _graphicsCanvas;
```

Comment:
Add a comment instead, this is implementation code (not API) so just use comments